public/modules/custom/grants_profile/src/Form/GrantsProfileFormBase.php

```diff
@@ -292,9 +255,77 @@ public function buildForm(array $form, FormStateInterface $form_state): array {
       '#limit_validation_errors' => [],
       '#submit' => ['Drupal\grants_profile\Form\GrantsProfileFormBase::formCancelCallback'],
     ];
+
     return $form;
   }
 
+  /**
+   * Remove attachments submit handler.
+   *
+   * @param array $form
+   *   Form.
+   * @param \Drupal\Core\Form\FormStateInterface $formState
+   *   Form state.
+   */
+  public static function removeAttachments(array &$form, FormStateInterface $formState): void {
+    $attachments = $form_state->get('attachments_to_remove');
```

Comment:
tästä tulee virhe:
`The website encountered an unexpected error. Please try again later.
Error: Call to a member function get() on null in Drupal\grants_profile\Form\GrantsProfileFormBase::removeAttachments() (line 271 of modules/custom/grants_profile/src/Form/GrantsProfileFormBase.php).`

näköjään toi formi väärin nimetty vaan. mut mä yritän saada nää pr:t selvittyä ni joudut ite fiksaa.